Makefile

```diff
@@ -111,6 +112,21 @@ build-query-linux:
 build-collector-linux:
 	CGO_ENABLED=0 GOOS=linux installsuffix=cgo go build -o ./cmd/collector/collector-linux ./cmd/collector/main.go
 
+.PHONY: docker
+docker: build_ui build-agent-linux build-collector-linux build-query-linux
+	cp -r jaeger-ui-build/build/ cmd/query/jaeger-ui-build
+	docker build -t $(DOCKER_NAMESPACE)/jaeger-cassandra-schema plugin/storage/cassandra/ ; \
```

Comment:
There's no "after" for this image :) It's used as a Kubernetes "Job": it will just run the shell script that generates the keyspace and will gracefully die afterwards. It's derived from the Cassandra image because it needs the `cqlsh` tools, so, I thought it would be easier (and save space) to just consume a image that I know existed there, but Cassandra itself will never start on this image.